package.json

```diff
@@ -12,9 +12,11 @@
     "atom": ">=1.0.0 <2.0.0"
   },
   "scripts": {
-    "build": "coffee grammars/dogescript.coffee > grammars/dogescript.cson"
+    "build": "./node_modules/coffescript/bin/coffee grammars/dogescript.coffee > grammars/dogescript.cson"
```

Comment:
When executed in the context on a package.json file, coffee should point to `./node_modules/coffescript/bin/coffee`, if coffeescript is installed as a local dependancy (which you have specified below). See https://docs.npmjs.com/misc/scripts#path

Reply:
^ I've corrected this is in your branch now :)

Reply:
Thanks a lot. I think i didn't need it cause I installed things globally. Thanks a bunch.